src/stores/models/Option.test.js

```diff
@@ -0,0 +1,303 @@
+import { observable } from 'mobx';
+import { Token, Phases, OracleStatus } from 'constants';
+import Option from './Option';
+const localStorageMock = {
+  getItem: jest.fn(),
+  setItem: jest.fn(),
+  clear: jest.fn(),
+};
+const navigatorMock = {};
+global.navigator = navigatorMock;
+global.localStorage = localStorageMock;
+
+const { QTUM, BOT } = Token;
+const { BETTING, VOTING, FINALIZING } = Phases;
+
+describe('stores/models/Option', () => {
+  const addr = {
+    address: 'qSu4uU8MGp2Ya6j9kQZAtizUfC82aCvGT1',
+    qtum: 2000,
+    bot: 100,
+  };
+  const app = observable({
+    sortBy: 'ASC',
+    wallet: { addresses: [addr] },
+    global: { syncBlockNum: 0 },
+    refreshing: false,
+    ui: { location: 0 },
+    eventPage: { selectedOptionIdx: 0 },
+  }); // mock the appstore
+  let store;
+
+  beforeEach(() => store = null);
+
+  it('Constructor 1: centralized oracle unconfirmed', () => {
```

Comment:
wrap all the constructor tests in a describe block. group tests by functions.
```
describe('Option', () => {
  describe('constructor()', () => {
  // all constructor tests
  };

  describe('toggleExpansion()', () => {
  // all toggleExpansion tests
  };
};
```

Reply:
Oh hey! yours is much more organized! will do it!